concepts/range-iteration/introduction.md

```diff
@@ -1,3 +1,97 @@
 # Introduction
 
-TODO: add introduction for range-iteration concept
+In Go, you can iterate a [slice](slices) using `for` but you could also use
+`range`. Range allows you to also iterate through a map or a
+[channel](channels). While iterating over a [channel](channels) is out of the
+scope for this exercise, you will find useful to iterate over a [map](maps).
```

Comment:
There are several links to outside resources. As per [the spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md#file-introductionmd):

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

I'd thus suggest to remove the links. You can totally have links in the `about.md` document though (it is even encourage there).

Reply:
> While iterating over a [channel](channels) is out of the
scope for this exercise

This document is presented outside the scope of an exercise, so it should not contain references to an exercise. I'll update the docs to make this clear.